src/CoreWCF.WebHttp/src/CoreWCF/Description/WebHttpBehavior.cs

```diff
@@ -0,0 +1,1356 @@
+                if (!WebMessageBodyStyleHelper.IsDefined(value))
+                {
+                    throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new ArgumentOutOfRangeException(nameof(value)));
+                }
+                _defaultBodyStyle = value;
+            }
+        }
+
+        public virtual WebMessageFormat DefaultOutgoingRequestFormat
+        {
+            get
+            {
+                return _defaultOutgoingRequestFormat;
+            }
+            set
+            {
+                if (!WebMessageFormatHelper.IsDefined(value))
+                {
+                    throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new ArgumentOutOfRangeException(nameof(value)));
+                }
+                _defaultOutgoingRequestFormat = value;
+            }
+        }
+
+        public virtual WebMessageFormat DefaultOutgoingResponseFormat
+        {
+            get
+            {
+                return _defaultOutgoingReplyFormat;
+            }
+            set
+            {
+                if (!WebMessageFormatHelper.IsDefined(value))
+                {
+                    throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new ArgumentOutOfRangeException(nameof(value)));
+                }
+                _defaultOutgoingReplyFormat = value;
+            }
+        }
+
+        public virtual bool HelpEnabled { get; set; }
+
+        public virtual bool AutomaticFormatSelectionEnabled { get; set; }
+
+        public virtual bool FaultExceptionEnabled { get; set; }
+
+        internal Uri HelpUri { get; set; }
+
+        // TODO: Can I remove this?
+        //protected internal string JavascriptCallbackParameterName { get; set; }
+
+        public virtual void AddBindingParameters(ServiceEndpoint endpoint, BindingParameterCollection bindingParameters)
+        {
+            // do nothing
+        }
+
+        public virtual void ApplyClientBehavior(ServiceEndpoint endpoint, ClientRuntime clientRuntime)
+        {
+            if (endpoint == null)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull(nameof(endpoint));
+            }
+
+            if (clientRuntime == null)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull(nameof(clientRuntime));
+            }
+
+            WebMessageEncodingBindingElement webEncodingBindingElement = endpoint.Binding.CreateBindingElements().Find<WebMessageEncodingBindingElement>();
+            if (webEncodingBindingElement != null && webEncodingBindingElement.CrossDomainScriptAccessEnabled)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new NotSupportedException(SR.CrossDomainJavascriptNotsupported));
+            }
+
+            _contractNamespace = endpoint.Contract.Namespace;
+
+            // TODO: Can I remove this?
+            //foreach (OperationDescription od in endpoint.Contract.Operations)
```

Comment:
Yes you can remove this commented out code.

Reply:
This was addressed in b03e198f117dedee05770fb621b326e3ba4b3e5a.